menas/ui/service/MonitoringService.js

```diff
@@ -53,6 +65,15 @@ var MonitoringService = new function() {
     });
   };
 
+  this.processDirSizes = function(oRun) {
+    let rawDirSize = oRun["controlMeasure"]["metadata"]["additionalInfo"]["raw_dir_size"];
+    let stdDirSize = oRun["controlMeasure"]["metadata"]["additionalInfo"]["std_dir_size"];
+    let publishDirSize = oRun["controlMeasure"]["metadata"]["additionalInfo"]["publish_dir_size"];
```

Comment:
Where do you take these keys from? I have a slight difference in my DB
```json
{
	"controlMeasure": {
		"metadata": {
			"sourceApplication": "TestDataMadeByHand",
			"country": "CZ",
			"historyType": "Snapshot",
			"dataFilename": "TestData.json",
			"sourceType": "Golden",
			"version": 1,
			"informationDate": "25-02-2019",
			"additionalInfo": {
				"conform_size_ratio": "24.80 %",
				"conform_username": "XXXXXXX",
				"std_application_id": "local-1558509454454",
				"conform_output_dir_size": "81840",
				"conform_records_failed": "20",
				"std_executors_num": "0",
				"conform_errors_count": "20",
				"std_username": "XXXXXXX",
				"std_records_succeeded": "0",
				"std_records_failed": "20",
				"conform_enceladus_version": "0.99.0-SNAPSHOT",
				"std_errors_count": "20",
				"std_output_dir_size": "329650",
				"std_input_dir_size": "67352",
				"conform_executors_num": "0",
				"std_enceladus_version": "0.99.0-SNAPSHOT",
				"conform_records_succeeded": "0",
				"conform_input_dir_size": "329949",
				"std_size_ratio": "489.44 %",
				"conform_application_id": "local-1558509701510"
			}
		}
	}
}
```

Reply:
Many thanks for this one. Added renaming of additionalInfo dir sizes in migrations 7078d3f